app/services/open_library_service.rb

```diff
@@ -0,0 +1,13 @@
+class OpenLibraryService
+  def self.execute(isbn)
+    include HTTParty
```

Comment:
Let's move this line to be  included at class level an not a  method  level

Reply:
Done